app/models/remote_authenticatable.rb

```diff
@@ -43,19 +43,31 @@ def initiate_auth(email, password)
         )
       end
 
-      def send_2fa(email, session_id, totp_code)
+      def respond_to_challenge(params)
+        challenge_name = params[:challenge_name]
```

Comment:
will this always be `SOFTWARE_TOKEN_MFA`?

Reply:
yes for now... until we introduce other methods/challenges..

Reply:
scratch that question, i realise you've also added the new password flow.